CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to ActivityPub for WordPress
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+Developers of all levels can help — whether you can barely recognize a filter (or don’t know what that means) or you’ve already authored your own plugins, there are ways for you to pitch in.
+
+## Create Bug Reports
+
+If you find a bug, please [file a GitHub issue](https://github.com/Automattic/wordpress-activitypub/issues/). If you want to prefix the title with a “Question:”, “Bug:”, or the general area of the application, that would be helpful, but by no means mandatory. If you have write access, add the appropriate labels.
+
+If you’re filing a bug, specific steps to reproduce are helpful. Please include the URL of the page that has the bug, along with what you expected to see and what happened instead.
+
+## Write and submit a patch
+
+If you'd like to fix a bug or make an enhancement, you can submit a Pull Request. Before you get started, you'll want to **[set up your development environment.](https://automattic.github.io/wordpress-activitypub/development-environment)**
+
+Once your development environment is ready, you can get started and [create your first Pull Request!](https://automattic.github.io/wordpress-activitypub/pull-request)
```

Comment:
Since GitHub offers markdown previews in the browser, I don't know if it's really necessary to have the GitHub Pages setup anymore? I think you could just link to the documents directly. It would simplify things quite a bit in your setup, you could focus on the contents and not really worry about the GitHub Pages website at all.

Reply:
@obenland @mattwiebe whats your view on that?

Reply:
Hmm… overall the less we have to do, the better!

Reply:
@mattwiebe the Jekyll stuff is already done, it is not about what you prefer!

Reply:
> whats your view on that?

Yes, I agree. I assume it's also the first place that new contributors will look for them.